src/udp.c

```diff
@@ -171,6 +171,10 @@ udp_bind ( int subsystem, const char *name,
   uc->rxsize               = rxsize;
   uc->txsize               = txsize;
 
+  if (bindaddr) {
+    uc->multicast = strncmp("239.", bindaddr, 4) == 0;
```

Comment:
This is not ipv6 ready

Reply:
Indeed and the worst part is that actually the multicast bit is set in the bind function just below that via reference, so no need to even bother with this, I removed it.